multimedia/views.py

```diff
@@ -10,3 +18,30 @@ def book_list(request):
 def book_detail(request, isbn13):
     book = Book.objects.get(isbn13=isbn13)
     return render(request, 'multimedia/book_detail.html', {'book': book})
+
+
+
+
+class MovieListView(TemplateView):
+    template_name = 'multimedia/movie_list.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['object_list'] = Movie.objects.all()
+        return context
+
+
+class MovieDetailView(TemplateView):
```

Comment:
At the moment, I think you may still used function based view or class based view. 

In Django:
> A view is a callable which takes a request and returns a response. This can be more than just a function.